sample-app/src/main/java/com/joinforage/android/example/pos/receipts/templates/txs/TxType.kt

```diff
@@ -60,37 +58,19 @@ enum class TxType(val title: String) {
             return UNKNOWN
         }
 
-        fun forPayment(payment: PosPaymentRequest): TxType {
-            if (spentSnap(payment)) {
-                return SNAP_PAYMENT
+        fun forPayment(transactionType: String?, fundingType: String): TxType {
+            if (transactionType == TransactionType.Withdrawal.value) {
+                return CASH_WITHDRAWAL
             }
-            if (spentEbtCash(payment) && withdrewEbtCash(payment)) {
+            if (transactionType == TransactionType.PurchaseWithCashBack.value) {
                 return CASH_PURCHASE_WITH_CASHBACK
             }
-            if (!spentEbtCash(payment) && withdrewEbtCash(payment)) {
-                return CASH_WITHDRAWAL
+            if (fundingType == FundingType.EBTSnap.value) {
+                return SNAP_PAYMENT
             }
-            if (spentEbtCash(payment) && !withdrewEbtCash(payment)) {
+            if (fundingType == FundingType.EBTCash.value) {
                 return CASH_PAYMENT
             }
-
-            return UNKNOWN
-        }
-
-        fun forRefund(payment: PosPaymentRequest): TxType {
```

Comment:
We weren't using this forRefund one anyway – we're using the forReceipt one for the refund screen right now